test/trento_web/views/health_view_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule TrentoWeb.HealthcheckViewTest do
```

Comment:
Rename this to `HeathViewTest` please

Reply:
Done, should this be updated on the [Wanda Repo](https://github.com/trento-project/wanda/commit/ccb2435b3587258c6606f0698cf5bf38d484ac9a#diff-34ccaab255f883f7e574ff2be41db4309d6c47cce7cef310a01337d23299fc23R1) also?

Reply:
PR for this change in Wanda: https://github.com/trento-project/wanda/pull/227